firebase/api/articles.js

```diff
@@ -53,8 +63,37 @@ export const getListOfAllArticlesByAlphabet = async () => {
   return groupedArticleData;
 };
 
-export const setNewArticle = async (articleObj) => {
+export const addNewArticle = async (articleObj) => {
```

Comment:
No need to use abbreviations. Also, any naming that has the structure as part of it, does not help with the understanding. The name should be implementation agnostic, at least most of the time.

So `articlePayload` could be more useful